libipl/p10/common.C

```diff
@@ -196,3 +198,22 @@ bool ipl_check_functional_master(void)
 
 	return true;
 }
+
+void  log_sbe_ffdc(struct pdbg_target *pib)
+{
+	uint8_t* ffdc = NULL;
+	uint32_t status = 0;
+	uint32_t ffdc_len = 0;
+
+        int ret = sbe_ffdc_get(pib, &status, &ffdc, &ffdc_len);
+	if (!ret)
+        ipl_log(IPL_INFO, "%s status:0x%x ffdc_len:0x%x : Data: ",
+		pdbg_target_path(pib), status, ffdc_len);
+        for (uint32_t i=0; i < ffdc_len; i++)
+		ipl_log(IPL_DEBUG, "%02x", ffdc[i]);
```

Comment:
Instead of printing all the bytes in a long row, you might want to wrap them after 16 or 32 bytes or so..  This will avoid long lines in the log.

Reply:
Ack